src/software/thunderscope/constants.py

```diff
@@ -317,3 +319,112 @@ class TrailValues:
 
     DEFAULT_TRAIL_LENGTH = 20
     DEFAULT_TRAIL_SAMPLING_RATE = 0
+
+
+class RobotControlType(Enum):
+    MOVE_X = 1
+    MOVE_Y = 2
+    ROTATE = 3
+    KICK = 4
+    CHIP = 5
+    KICK_POWER = 6
+    DRIBBLER_SPEED = 7
+    DRIBBLER_ENABLE_1 = 8
+    DRIBBLER_ENABLE_2 = 9
+
+
+# TODO: come up with less ambiguous/weird name
+class HandheldDeviceConfigKeys(Enum):
+    CODE = 1
+    MAX_VALUE = 2
+
+
+# nomenclature:
+
+
+@dataclass
+class DeviceKeyEvent:
+    """
+    This dataclass holds the code for a "EV_KEY" input event
+    """
+
+    event_code: int
+
+
+@dataclass
+class DeviceAbsEvent:
+    """
+    This dataclass holds the code and max value for an "EV_ABS" input event
+
+    """
+
+    event_code: int
+    max_value: float
+
+
+@dataclass
+class DeviceConfig:
+    move_x: DeviceAbsEvent
+    move_y: DeviceAbsEvent
+    move_rot: DeviceAbsEvent
+    kick: DeviceKeyEvent
+    chip: DeviceKeyEvent
+    chicker_power: DeviceAbsEvent
+    dribbler_speed: DeviceAbsEvent
+    primary_dribbler_enable: DeviceAbsEvent
+    secondary_dribbler_enable: DeviceAbsEvent
+
+
+class HandheldDeviceConstants:
```

Comment:
I don't know if HandheldDeviceConstants is a good name for this class -- consider `DiagnosticsControlConstants` or something similar?